### PR TITLE
How to extend grid with Javascript extensions

### PR DESCRIPTION
**Pull request by zuk3975**
Added tutorial that explains how to use Javascript extensions for grid.

### CODE REVIEW

**Comment by zuk3975** [comment]
@matks my first ps tutorial :smile:

**Comment by matks** [comment]
> @matks my first ps tutorial 😄

File: src/content/1.7/development/components/grid/tutorials/extend-grid-with-javascript.md [hunk 1]
```diff
@@ -3,4 +3,80 @@ title: How to extend a Grid with Javascript extensions?
 weight: 4
 ---
 
-> Do you know how to do it? Please feel free to contribute to the docs!
\ No newline at end of file
+# Extend a grid with Javascript extensions. Preparation
+Javascript grid extensions supplements grid functionality and enhances user experience.
```
Comment (matks) [inline src/content/1.7/development/components/grid/tutorials/extend-grid-with-javascript.md]:
```suggestion
Javascript grid extensions supplement grid functionality and enhance user experience.
```

File: src/content/1.7/development/components/grid/tutorials/extend-grid-with-javascript.md [hunk 2]
```diff
@@ -3,4 +3,80 @@ title: How to extend a Grid with Javascript extensions?
 weight: 4
 ---
 
-> Do you know how to do it? Please feel free to contribute to the docs!
\ No newline at end of file
+# Extend a grid with Javascript extensions. Preparation
```
Comment (matks) [inline src/content/1.7/development/components/grid/tutorials/extend-grid-with-javascript.md]: `Preparation` ?

File: src/content/1.7/development/components/grid/tutorials/extend-grid-with-javascript.md [hunk 3]
```diff
@@ -3,4 +3,80 @@ title: How to extend a Grid with Javascript extensions?
...
+Javascript grid extensions supplements grid functionality and enhances user experience.
+For example, most grids contains SortingExtension which provides sorting by columns functionality, 
+BulkActionCheckboxExtension which enables interactive selection of bulk actions and so on.
+
+In this tutorial we use an example of taxes grid to show how to use extensions.
```
Comment (matks) [inline src/content/1.7/development/components/grid/tutorials/extend-grid-with-javascript.md]:
```suggestion
In this tutorial we use a grid that display taxes items to show how to use extensions.
```

File: src/content/1.7/development/components/grid/tutorials/extend-grid-with-javascript.md [hunk 4]
```diff
@@ -3,4 +3,80 @@ title: How to extend a Grid with Javascript extensions?
...
+To start we will need to have a plain grid already created in php side
+(you can find more docs about grid here https://devdocs.prestashop.com/1.7/development/components/grid).
+After following some guides we should be ready. Our grid looks like this:
```
Comment (matks) [inline src/content/1.7/development/components/grid/tutorials/extend-grid-with-javascript.md]:
```suggestion
After following the Grid documentation directions you should now have a fully working php grid. The twig template should look like this:
```

**Comment by matks** [comment]
Thank you @zuk3975

**Pull request merged**